src/FSharpPlus/Data/NonEmptyList.fs

```diff
@@ -178,6 +184,9 @@ type NonEmptyList<'t> with
     [<EditorBrowsable(EditorBrowsableState.Never)>]
     static member Map (x: NonEmptyList<'a>, f: 'a->'b) = NonEmptyList.map f x
     
+    [<EditorBrowsable(EditorBrowsableState.Never)>]
+    static member MapI(x: NonEmptyList<_>, f) = NonEmptyList.mapi f x
```

Comment:
If you look into Indexable.fs you'll see this method should be called `MapIndexed`.
For the sake of consistent formatting please leave a space between the method name and the opening parens for the arguments. I know in C# you normally write them together, but here's not the case.